lighter/utils/importing.py

```diff
@@ -0,0 +1,43 @@
+import sys
```

Comment:
Can we rename this file? Naming is inconsistent with the rest.

Maybe `dynamic_imports.py` ?

Reply:
Cool, done